cli/cmd/helm/helmInstall.go

```diff
@@ -223,6 +233,15 @@ func appendDBURIValues(dbURI, name string, helmValues string) string {
 	return helmValues
 }
 
+func appendPrometheusHAValues(helmValues string) string {
```

Comment:
is this ever tested anywhere?

Reply:
Yup, The HA tests run tobs install with `--enable-prometheus-ha` which invokes this func.

Reply:
Also, the PR level tests i.e. `e2e-tests` are also performed on tobs HA setup to make sure all tobs CLI features work in HA mode. 
Now no `values.yaml` contain HA spec by default. HA deployment will be triggered only by using `--enable-promtheus-ha` flag.